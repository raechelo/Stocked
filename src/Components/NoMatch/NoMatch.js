import React from 'react';
import { Link } from 'react-router-dom';
import cuttingboard from '../../assets/cuttingboard.png';

export const NoMatch = () => {
  return (
    <div className="NoMatch">
      <h3>Sorry! That page doesn't exist.</h3>
      <img src={cuttingboard} />
      <Link className="back-btn" to='/'>
        <button>Go Back</button>
      </Link>
    </div>
  )
}

export default NoMatch;