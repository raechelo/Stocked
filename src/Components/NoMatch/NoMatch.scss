.NoMatch {
  text-align: center;

  h3 {
    font-family: $fancy-font;
    font-size: 2rem;
  }

  button {
    background-color: $fave;
    border: 2px solid $accent;
    border-radius: 3px;
    color: white;
    display: block;
    font-family: $body-font;
    font-size: 1rem;
    margin: auto;
    text-decoration: none;

    &:hover {
      background-color: $accent;
      border: 2px solid $fave;
      text-decoration: underline;
    }
  }

  .back-btn {
    text-decoration: none;

    &:hover {
      text-decoration: underline;
    }
  }
}