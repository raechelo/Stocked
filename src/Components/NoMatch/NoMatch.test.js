import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from './NoMatch';

describe('NoMatch', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NoMatch />);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});