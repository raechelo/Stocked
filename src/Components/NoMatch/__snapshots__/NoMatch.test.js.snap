// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match the snapshot with all the data passed in 1`] = `ShallowWrapper {}`;
